File: solution/1500-1599/1526.Minimum Number of Increments on Subarrays to Form a Target Array/Solution.cs
public class Solution {
    public int MinNumberOperations(int[] target) {
        int f = target[0];
        for (int i = 1; i < target.Length; ++i) {
            if (target[i] > target[i - 1]) {
                f += target[i] - target[i - 1];
            }
        }
        return f;
    }
}

File: solution/1500-1599/1526.Minimum Number of Increments on Subarrays to Form a Target Array/Solution.rs
impl Solution {
    pub fn min_number_operations(target: Vec<i32>) -> i32 {
        let mut f = target[0];
        for i in 1..target.len() {
            if target[i] > target[i - 1] {
                f += target[i] - target[i - 1];
            }
        }
        f
    }
}
